apps/condo/domains/common/components/Card/Card.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Card as DefaultCard, CardProps as DefaultCardProps } from 'antd'
+
+import { CardWrapper, CardWrapperProps } from './CardWrapper'
+
+type CardProps = CardWrapperProps & DefaultCardProps
+
+export const Card: React.FC<CardProps> = ({ disabled, children, ...otherCardProps }) => {
```

Comment:
Are you sure you need a new component?

It seems that the same cards are used in analytics

Reply:
This card is taken from the miniapps and placed in the common folder.
Only need to replace the card miniapps with these from the common folder